channels/rdpecam/server/camera_device_main.c

```diff
@@ -685,7 +685,7 @@ static wStream* device_server_packet_new(size_t size, BYTE version, BYTE message
 {
 	wStream* s;
 
-	WINPR_ASSERT(size > 0);
+	WINPR_ASSERT(size + CAM_HEADER_SIZE > 0);
```

Comment:
shouldn´t this just be removed? `CAM_HEADER_SIZE` is defined `> 0` anyway

Reply:
Removed the assertion in the last push.